config/migrations/2024/20240313000000-installatievergadering-statussen.ttl

```diff
@@ -0,0 +1,45 @@
+@prefix rdf:  <http://www.w3.org/1999/02/22-rdf-syntax-ns#>.
+@prefix skos:	<http://www.w3.org/2004/02/skos/core#>.
+@prefix rdfs:	<http://www.w3.org/2000/01/rdf-schema#>.
+@prefix mu:   <http://mu.semte.ch/vocabularies/core/>.
+@prefix conceptschemes: <http://lblod.data.gift/concept-schemes/>.
+@prefix ext: <http://mu.semte.ch/vocabularies/ext/>.
+@prefix ivs: <http://data.lblod.info/id/InstallatievergaderingStatus/>.
+
+
+# Installatievergadering status concept scheme
+conceptschemes:installatievergadering-status
+  rdf:type 
+    skos:ConceptScheme,
+	  skos:Concept,
```

Comment:
this should not be a skos:Concept

Reply:
Ah damn, I also did it like this in decretaal-bestuursorgaan-codes.ttl